### DISCUSSION TITLE
About the error 'XMinioStorageFull'

### DISCUSSION THREAD

**Discussion by Zzx8023Jm** [minio/minio#17045] [Q&A] [answered]
I have  a similar question like  this. https://github.com/minio/minio/issues/6795
I use minio to upload some pictures. At first ,everything is okay.However, it began to  throw 'XMinioStorageFull' frequently after runing a few month. The current behaviour is ,when i try to upload 100 pictures,maybe 70 of them will not succeed.
I thought maybe the reason is that the number of objects in one bucket is too large  at first.So i  created a new bucket and uploaded  new pictures to it .But the question seemed to not be resolved.
The root of minio is /data,and the disk status is 
[root@localhost training-center-service]# df -h
文件系统                 容量  已用  可用 已用% 挂载点
devtmpfs                 7.6G     0  7.6G    0% /dev
tmpfs                    7.6G     0  7.6G    0% /dev/shm
tmpfs                    7.6G  795M  6.8G   11% /run
tmpfs                    7.6G     0  7.6G    0% /sys/fs/cgroup
/dev/mapper/centos-root  438G  138G  300G   32% /
/dev/sda2               1014M  169M  846M   17% /boot
/dev/sda1                200M   12M  189M    6% /boot/efi
/dev/sdb1                 22T  4.0T   17T   20% /data
tmpfs                    1.6G   12K  1.6G    1% /run/user/42
tmpfs                    1.6G     0  1.6G    0% /run/user/0
I will appreciate for someone to help.

**Comment by harshavardhana** [comment, marked as answer]
Do df -ih and are you using ext4 ?

**Reply by Zzx8023Jm** [reply]
Yes .
/dev/sdb1               ext4     22869170132 4192026388 17529094032   20% /data

**Reply by Zzx8023Jm** [reply]
Thank you for answering.I just found the  reason in this issure https://github.com/minio/minio/issues/16143.
Really appreciate for your reply.

**Reply by harshavardhana** [reply]
You need to move to XFS or enable ext4 large directory support.

**Reply by Zzx8023Jm** [reply]
Okay.Thanks a lot .
